idseq/__init__.py

```diff
@@ -125,95 +135,126 @@ def main():
         type=int,
         help='Memory requirement in MB')
     parser.add_argument(
-        '--host-id',
-        metavar='value',
-        type=int,
-        help='Host Genome Id')
+        '--host-id', metavar='value', type=int, help='Host Genome Id')
     parser.add_argument(
-        '--host-genome-name',
-        metavar='name',
-        type=str,
-        default="Human",
-        help='Host Genome Name')
-    parser.add_argument(
-        '--job-queue',
-        metavar='name',
-        type=str,
-        help='Job Queue')
+        '--job-queue', metavar='name', type=str, help='Job Queue')
     parser.add_argument(
         '--uploadchunksize',
         metavar='value',
         type=int,
         default=DEFAULT_MAX_PART_SIZE_IN_MB,
         help='Break up uploaded files into chunks of this size in MB')
-
+    parser.add_argument(
+        '--accept-all',
+        action='store_true',
+        help='Use this argument to automatically accept confirmation messages')
     args = parser.parse_args()
 
+    # Use https://idseq.net by default
+    print("")
+    if not args.url:
+        args.url = "https://idseq.net"
+
+    # Prompt the user for missing fields
+    if not args.email:
+        args.email = required_input("Enter your IDseq account email: ")
+    if not args.token:
+        args.token = required_input("Enter your IDseq authentication token ("
+                                    "found in instructions): ")
+    if not args.project:
+        args.project = required_input("Enter the project name: ")
+    if not args.bulk:
+        if not args.sample_name:
+            inp = input("Enter the sample name (or press Enter to "
+                        "use bulk mode): ".ljust(35))
+            if inp is '':
+                args.bulk = "."  # Run bulk auto-detect on the current folder
+            else:
+                args.sample_name = inp
+                if not args.r1:
+                    args.r1 = required_input(
+                        "Enter the first file (first in a "
+                        "paired-end run or sole file in a "
+                        "single-end run): ")
+                if not args.r2:
+                    r2 = input(
+                        "Enter the second paired-end file if applicable (or "
+                        "press Enter to skip): ")
+                    if r2 != '':
+                        args.r2 = r2
+    while args.host_genome_name not in [
```

Comment:
Yeah the host genome is what's filtered out in the host filtering step, like "what not to look at" since we just want to retain the pathogens. Doesn't affect what DB to align against (they all compare with NT and NR) and doesn't necessarily affect the default background.